package.json

```diff
@@ -44,7 +44,7 @@
     "fastboot-app-server": "1.0.1",
     "hammerjs": "2.0.8",
     "js-cookie": "2.2.0",
-    "jwplayer-fandom": "1.1.1",
```

Comment:
remember to change it to release version

Reply:
👍